Assets/MixedRealityToolkit.Services/InputSimulation/Editor/InputSimulationService.cs

```diff
@@ -13,9 +13,10 @@ namespace Microsoft.MixedReality.Toolkit.Input
         typeof(IMixedRealityInputSystem),
         SupportedPlatforms.WindowsEditor,
         "Input Simulation Service",
-        "Profiles/DefaultMixedRealityInputSimulationProfile.asset", 
+        "Profiles/DefaultMixedRealityInputSimulationProfile.asset",
         "MixedRealityToolkit.SDK")]
-    public class InputSimulationService : BaseInputDeviceManager, IInputSimulationService
```

Comment:
>IInputSimulationService [](start = 66, length = 23)

Just for my own understanding, what was the reason for removing the IInputSimulationService from the InputSimulationService implementation?

From a casual reader it seems like this implementation should implement the interface, so I'm curious why it's being removed.

Reply:
This looks like a confused merge due to the doc link attribute. I'll put it back.